pkg/plugins/plugins.go

```diff
@@ -221,7 +223,17 @@ func Get(name, imp string) (*Plugin, error) {
 
 // Handle adds the specified function to the extpointHandlers.
 func Handle(iface string, fn func(string, *Client)) {
-	extpointHandlers[iface] = fn
+	handlers, ok := extpointHandlers[iface]
+	if !ok {
+		handlers = []func(string, *Client){fn}
+	}
+
+	handlers = append(handlers, fn)
+	extpointHandlers[iface] = handlers
+	pa, _ := GetAll(iface)
+	for _, p := range pa {
+		p.activated = false
```

Comment:
This is required because we want to make sure p.activated is reset to false if a 2nd handler gets registered later than the first handler gets the activation call.

Once activated, plugin never gives the callback. Hence whenever a new handler is added, we have to reset the flag so that they can receive the callback.